packages/remix-server-runtime/package.json

```diff
@@ -17,15 +17,15 @@
   "module": "dist/esm/index.js",
   "dependencies": {
     "@remix-run/router": "1.5.0",
-    "@types/cookie": "^0.4.0",
-    "@types/react": "^18.0.15",
     "@web3-storage/multipart-parser": "^1.0.0",
     "cookie": "^0.4.1",
     "set-cookie-parser": "^2.4.8",
     "source-map": "^0.7.3"
   },
   "devDependencies": {
     "@remix-run/web-file": "^3.0.2",
+    "@types/cookie": "^0.4.0",
+    "@types/react": "^18.0.15",
```

Comment:
We're exporting types that use `@types/react`, so we should keep this one in `dependencies`

It can be removed if we merge #5713 though